src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,7 +5,12 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
```

Comment:
![image](https://user-images.githubusercontent.com/12864128/215261605-2115cbb5-1744-4c27-b00d-21ef71e1e431.png)
Add some fields to your Machine subclasses